### ISSUE TITLE
Block entire instance instead of just community

### ISSUE THREAD

**Issue by krisezra87** [dessalines/jerboa#1279]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

A cursory search did not turn this up in open or closed issues.  This seems like something 

It would be excellent to block by regex or block entire instances.  There is literally nothing on ani.social / yiffit / hexbear / etc that I ever want to see and I want to browse all by default to find new communities. 

Is this cooking and just not in the issues list?

**Comment by MV-GH** [comment]
This will be support in 0.19 by lemmy and will then be supported by Jerboa for 0.19 instances.

**Comment by krisezra87** [comment]
I agree.  I actually switched to Voyager just so I can have it in-app.  Reopened.

**Comment by mathieures** [comment]
Thank you for pointing this out! I was expecting this kind of button to be on a community page after clicking on the three dots, but I'm fine with having it on a post instead.

Maybe I missed it in the changelog, but if it's really not there I would say it would be nice to mention it in there (unless it's too late now).

**Comment by MV-GH** [comment]
We can easily add it there too, just make an issue for it
